test/stripe.spec.ts

```diff
@@ -156,30 +156,6 @@ describe('Stripe Module', function() {
     });
   });
```

Comment:
Wasn't there a test where we reached into Stripe and set the _platformFunctions field directly?

Reply:
It's the uname tests: https://github.com/stripe/stripe-node/blob/663f7dbc1c67b968358d8226b78d8f19b45884d0/test/stripe.spec.ts#L271

But in `stripe.common` we still access this implementation through `_platformFunctions` on the instance: https://github.com/stripe/stripe-node/pull/1679/files/ef4162c1b220a166df91bd7151896128188cdb0f#diff-9ed8c83003262337d25a9d027e4591a600743fca72db01a1b1ef02ff084fae74R347

Reply:
Right, but with your new factory function you can create a new instance of stripe and provide `getMockPlatformFunctions()..` as parameter.

Reply:
Updated in https://github.com/stripe/stripe-node/pull/1679/commits/b43f1fb28dac6ae470e08feedc998e1aafbe1eee